src/backend/launcher.ts

```diff
@@ -545,7 +537,9 @@ function setupWineEnvVars(gameSettings: GameSettings, gameId = '0') {
     }
     case 'proton':
       ret.STEAM_COMPAT_CLIENT_INSTALL_PATH = steamInstallPath
+      ret.WINEPREFIX = winePrefix
       ret.STEAM_COMPAT_DATA_PATH = winePrefix
+      ret.PROTONPATH = dirname(gameSettings.wineVersion.bin)
```

Comment:
If you choose not to set this, it will cause umu download the latest UMU-Proton and setting PROTONPATH=GE-Proton, will cause it to download the latest GE-Proton. 

When umu is configured as the runtime, perhaps another option can be added to the dropdown list as the first option (e.g., UMU-Latest)?